docs/source/alfacase_definitions/AnnulusDescription.txt

```diff
@@ -18,8 +18,8 @@
 
             has_annulus_flow: boolean
             has_annulus_flow: number
-            pvt_model: string # optional
-            initial_conditions: \ :class:`initial_conditions_description_schema <InitialConditionsDescription>`\ # optional
-            gas_lift_valve_equipment: 
-                string: \ :class:`gas_lift_valve_equipment_description_schema <GasLiftValveEquipmentDescription>`\ # optional
+            pvt_model: string  # optional
+            initial_conditions: \ :class:`initial_conditions_description_schema <InitialConditionsDescription>`\  # optional
+            gas_lift_valve_equipment:  # optional
+                string: \ :class:`gas_lift_valve_equipment_description_schema <GasLiftValveEquipmentDescription>`
```

Comment:
I think we did add the escape at the end back. I can't find the code.

```
                    # Remove the last character if it's a backslash otherwise the line break will be avoided.
                    if line_content.endswith("\\"):
                        line_content = line_content[:-1]
```

OK, since the `# optional` moved the last escape is removed.

Reply:
Just to confirm, it is rendering correctly:

![image](https://user-images.githubusercontent.com/3778802/116729147-35b68800-a9bd-11eb-93a6-d16cc8642f9b.png)

https://alfasim-sdk--158.org.readthedocs.build/en/158/alfacase/api_reference.html#alfasim_sdk.AnnulusDescription

Reply:
Duplicities handled in another PR: https://github.com/ESSS/alfasim-sdk/pull/164